src/site/_data/postTags.js

```diff
@@ -204,6 +204,10 @@ const tags = {
   'virtual-reality': {
     title: 'Virtual Reality',
   },
+  'web-animations': {
+    title: 'Web Animations',
+    description: 'Our latest news, updates, and stories about Web Animations.',
+  },
```

Comment:
```suggestion
  animations: {
    title: 'Animations'
  },
```
I don't think we need a tag dedicated to the Web Animations API. A tag for animations would be good to have, though. Please move this to preserve the alphabetical ordering of the tags.